sfcc_cartridge/int_fast_sfra/cartridge/static/default/js/fastProduct.js

```diff
@@ -11,26 +11,39 @@ $(document).ready(function () {
         checkoutFastButton.removeClass('fast-checkout');
     }
 
+    // following code will exucute when click on quickview button which having class 'quickview'
+    $('body').on('click', '.quickview', function (e) {
+        e.preventDefault();
+        
+        setTimeout(function() {
+            initQuickviewFastBtn();
+        }, 1000);
```

Comment:
@isafranyos delay here because quickview popup (which have fast checkout button) take some time to show and if we add click event without delay then fast button event listener wont work as button not available on that time.